content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -3,14 +3,124 @@ title: Configure the Datadog Tracing Library
+: **Default**: Varies by SDK, environment, and runtime. Please read more in the specific configuration page above for your chosen language <br>
+**Supported Input**: A valid full or relative directory path that exists on the system<br>
+**Caveats**: Not supported in Java, Node.js, Ruby, Python<br>
+**Description**: Specifies the directory where tracer log files should be routed. If the directory does not exist, the SDK falls back to its default diagnostic logging method.
+
+### Agent
+
+`DD_TRACE_AGENT_URL`
+: **Default**: `http://localhost:8126` <br>
+**Supported Input**: A string representing an HTTP or UDS url <br>
+**Description**: The URL for connecting the tracer to the Datadog agent. Valid URL schemas include `http://` and `unix://` (UNIX Domain Sockets). This value takes precedence over `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` if set.
+
+### Unified Service Tagging
+
+`DD_VERSION`
+: **Default**: `null`<br>
+**Supported Input**: A string representing an application version<br>
+**Caveats**:<br/>
+  - Node.js automatically sets the version number from package.json
+  - [Inferred services][3] will not set version tags
+**Description**: Adds a `version` tag to all spans
+
+`DD_SERVICE`
+: **Default**: `null`, the SDK will try to automatically determine a service name<br>
+**Supported Input**: A string representing an application service name <br>
+**Description**: Sets the default service name used for most spans. SDKs may set a different service name for inferred services. Integration spans may use their own default names, which can differ from the value specified in `DD_SERVICE`
+
+`DD_ENV`
+: **Default**: `null` <br>
+**Supported Input**: A string representing an application environment name (for example, `prod`, `dev`) <br>
+**Description**: Adds an environment tag to all spans generated by the tracer instance.
+
+### Integrations
+
+`DD_TRACE_HTTP_CLIENT_ERROR_STATUSES`
+: **Default**: `400-499` <br>
+**Supported Input**: A comma-separated string of the form `from-to`, where `from` and `to` are integers. Singular values are also accepted (for example, `400-403,405,410-499`). <br>
+**Caveats**: Not supported in Node.js<br>
+**Description**: Defines the inclusive range of status codes to be considered as errors on automatically collected HTTP client spans. Only the values within the specified range are considered errors.
+
+`DD_TRACE_HTTP_SERVER_ERROR_STATUSES`
+: **Default**: `500-599` <br>
+**Supported Input**: A comma-separated string of the form `from-to`, where `from` and `to` are integers. Singular values are also accepted (for example, `400-403,405,410-499`). <br>
+**Caveats**: Not supported in Node.js<br>
+**Description**: Defines the inclusive range of status codes to be considered errors on `http.server` span kinds. Only the values within the specified range are considered errors.
+
+`DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING`
+: **Default**: `true` <br>
+**Supported Input**: Boolean <br>
+**Caveats**: Not supported in Node.js<br>
```

Comment:
We're disabling this by default in Go (but can be enabled with DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING=true)